src/renderer/modals/SettingsAccount/AccountSettingRenderBody.js

```diff
@@ -262,12 +262,12 @@ export default ConnectedAccountSettingRenderBody;
 
 export const Container: ThemedComponent<{}> = styled(Box).attrs(() => ({
   flow: 2,
-  justify: "space-between",
```

Comment:
i think it was renamed `justifyContent` , but your solution works too

Reply:
that design system opportunity will be perfect to decide if we actually want to get rid of the <Box /> paradigm

Reply:
hell yeah